### ISSUE TITLE
Mousewheel support (scrolling)

### ISSUE THREAD

**Issue by White-Tiger** [White-Tiger/T-Clock#40]
It would be nice if T-Clock could support and handle the mouse wheel. Eg. scroll on the clock to adjust volume up or down.

Possible actions:
* Volume up/down
* create instant alarm in ±5 minutes _(scroll to increase time, stop for ~3s to save and run it)_
* normal click actions
* clock profile switch _(once T-Clock receives profiles)_

also add modifier keys, such as Shift, Ctrl, Alt, RShift, RCtrl, RAlt (and combinations of those)

**Comment by MKaprielian** [comment]
I agree these would be great Enhancements.  I would use several them as soon as they become available.
